.tekton/cluster-observability-operator-1-2-push.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
               ".tekton/cluster-observability-operator-1-2-push.yaml".pathChanged() ||
               "Dockerfile.obo".pathChanged() ||
               "observability-operator/***".pathChanged())
+    build.appstudio.openshift.io/build-nudge-files: bundle-patches/render_templates
```

Comment:
@lihongyan1 @danielmellado this is added by the latest version of `customize-pipeline.sh`. Should I leave it?

Reply:
We should keep this.